runtime/errors.rs

```diff
@@ -153,12 +154,21 @@ pub fn get_nix_error_class(error: &nix::Error) -> &'static str {
   }
 }
 
+fn get_canvas_error(e: &CanvasError) -> &'static str {
```

Comment:
Making this part of deno_canvas might be better considering implementors like Deploy can re-use the same function?

Reply:
@satyarohith i am currently going through all exts, and this is a temporary solution until we can land a error rework PR in deno_core (which is stuck by the fact that cli would need a massive PR, so am splitting things into multiple steps); the problem also is a lot of the exts need access to some of the get_error_class functions defined in other exts or in runtime (ie `get_io_error_class`)